source/bases/Win32Service.c

```diff
@@ -164,27 +167,33 @@ static int Service_SessionChange(DWORD sessionId, DWORD eventType)
 {
     PyThreadState *threadState;
     PyObject *result;
+    char *error_message = NULL;
```

Comment:
Similar comments apply here. These changes also do not seem to be relevant for the concerns initially raised: namely, memory leaks.

Reply:
I call this "other types of leaks", because the thread is aquired but not released.